lib/handlers/router-entities/route-caching/cached-routes-configuration.ts

```diff
@@ -188,17 +134,17 @@ export const CACHED_ROUTES_CONFIGURATION: Map<string, CachedRoutesStrategy> = ne
       tradeType: TradeType.EXACT_INPUT,
       chainId: ChainId.MAINNET,
       buckets: [
-        new CachedRoutesBucket({ bucket: 0.0025, cacheMode: CacheMode.Tapcompare, maxSplits: 1 }),
-        new CachedRoutesBucket({ bucket: 0.006, cacheMode: CacheMode.Tapcompare, maxSplits: 1 }),
-        new CachedRoutesBucket({ bucket: 0.025, cacheMode: CacheMode.Tapcompare, maxSplits: 1 }),
-        new CachedRoutesBucket({ bucket: 0.06, cacheMode: CacheMode.Tapcompare, maxSplits: 1 }),
-        new CachedRoutesBucket({ bucket: 0.25, cacheMode: CacheMode.Tapcompare, maxSplits: 2 }),
-        new CachedRoutesBucket({ bucket: 0.6, cacheMode: CacheMode.Tapcompare, maxSplits: 2 }),
-        new CachedRoutesBucket({ bucket: 1, cacheMode: CacheMode.Tapcompare, maxSplits: 2 }),
-        new CachedRoutesBucket({ bucket: 2, cacheMode: CacheMode.Tapcompare, maxSplits: 2 }),
-        new CachedRoutesBucket({ bucket: 3, cacheMode: CacheMode.Tapcompare, maxSplits: 2 }),
-        new CachedRoutesBucket({ bucket: 4, cacheMode: CacheMode.Tapcompare }),
-        new CachedRoutesBucket({ bucket: 5, cacheMode: CacheMode.Tapcompare }),
+        new CachedRoutesBucket({ bucket: 0.0025, cacheMode: CacheMode.Tapcompare, withLastNCachedRoutes: 3 }),
```

Comment:
do all these `CachedRoutesBucket` now have a maxSplits of 0??

Reply:
yeah, but maxSplits of 0 means that any number of splits is allowed :) Another configuration change.

I originally introduced the `maxSplits` filter to remove low liquidity routes that had fragmented liquidity, but I think that filter might not be necessary when we look at the N previous best routes